lib/simplifyExpression/index.js

```diff
@@ -15,5 +15,4 @@ function simplifyExpressionString(expressionString, debug=false) {
   return [];
 }
 
-module.exports = simplifyExpressionString;
-
+module.exports = simplifyExpressionString;
```

Comment:
all files should end in exactly one new line

you had two before, which is why I commented

but now wherever you see the 🚫 on github you should add another new line at the end